RELEASE_PROCESS.md

```diff
@@ -259,6 +259,8 @@ spelled with complete minutiae.
       press the "Edit Release" button.  Change the name to the form `vX.Y.Z`
    1. If this is a release candidate be certain to click the pre-release
       checkbox at the bottom of the page.
+   1. If this is the highest version release be certain to click the
```

Comment:
Maybe "If the new release will be the latest version released, be certain to..."

Reply:
Reworded the text. Thanks for the feedback.